v2/dtos/notification.go

```diff
@@ -27,6 +29,22 @@ type Notification struct {
 	Status             string   `json:"status,omitempty" validate:"omitempty,oneof='NEW' 'PROCESSED' 'ESCALATED'"`
 }
 
+// NewNotification creates and returns a Notification DTO
+func NewNotification(labels []string, category, content, contentType, description, sender, severity, status string) Notification {
```

Comment:
Per previous discussions on factory methods, the parameters should be just the required properties. Having `labels` or `category` make this strange... but others such as `description` , `status`, etc. should be removed.

Reply:
Modified the function signature to
`func NewNotification(labels []string, category, content, sender, severity string) Notification`